### DISCUSSION TITLE
About some go packages

### DISCUSSION THREAD

**Discussion by solracsf** [juicedata/juicefs#2691] [General]
[mattn/go-sqlite3](https://github.com/juicedata/juicefs/blob/main/go.mod#L40) is using a wrong version instead of [supported ones](https://github.com/mattn/go-sqlite3/tags)
_(see https://github.com/mattn/go-sqlite3#go-sqlite3 about this mistake)_

[golang/protobuf](https://github.com/juicedata/juicefs/blob/main/go.mod#L25) is deprecated: Use the "`google.golang.org/protobuf`" module instead.

**Comment by SandyXSD** [comment]
Thank you, both are updated.

**Reply by solracsf** [reply]
Can't these be managed using Github Dependabot?
https://gist.github.com/magnetikonline/6f215db058e327905bce66c37f92426c

**Reply by zhijian-pro** [reply]
Looks great, we'll try it out.
